File: 30DaysOfCode/Day 25 Running Time and Complexity/Solution.py
import math

def isPrime(n):
    for i in range(2, int(math.sqrt(n))+1 ):
        if n%i == 0:
            return False
    return True

t = int(input())
for _ in range(t):
    inp = int(input())
    if inp>=2 and (isPrime(inp)):
        print('Prime')
    else:
        print('Not prime')

File: 30DaysOfCode/Day 25 Running Time and Complexity/Solution.java
import java.io.*;
import java.util.*;

public class Solution {

    public static boolean isPrime(int n) {
        for (int i = 2; i <= Math.sqrt(n); i++) {
            if (n % i == 0) {
                return false;
            }
        }
        return true;
    }
    
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int t = input.nextInt();

        for (int i = 0; i < t; i++) {
            int n = input.nextInt();

            if (n >= 2 && isPrime(n))
                System.out.println("Prime");
            else System.out.println("Not prime");
        }
    }
}